website/TRANSLATIONS.md

```diff
@@ -0,0 +1,39 @@
+# Contributing Guidelines for our docs
+
+Are you a native speaker of a language other than English? We'd love to have your help!
+
+## How to contribute
+
+(Read the [contribution guidelines](../CONTRIBUTING.md) first for general information about contributing to this project.)
+
+### Initial Translation
+
+If the docs are not available in your language yet and you would like to translate them, you can do so by following these steps:
+
+1. Copy the `src/content/docs` directory to a new directory (exclude all other translated code, e.g. `ja`, `pt-br`, etc.) with the name of your language, e.g. `src/content/docs/de` for German. (If you're not sure what the language code is, you can find it [here](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).)
```

Comment:
I think this isn't a good suggestion, mainly because starlight works in a different way. A better suggestion would be to configure the website first, and then progressively add the translated pages.

If a translated page is missing, starlight will take care of it. There's no need to copy all the content.

Reply:
That's correct, I might update it to "copy accordingly" or "what you copy, what you translate (WYCWYT)"